### ISSUE TITLE
does this tool work?

### ISSUE THREAD

**Issue by cyrfer** [SE7ENSKY/group-css-media-queries#12]
I tried this project and I saw no change in the output. A new file was created, but the new file still suffered from issues of the source file. I have a ton of media queries from Less transpiling to CSS that I want to group for proper selection and optimization.

Does this project actually work?

Here were my steps to test it:
- I installed via: `npm i --save-dev group-css-media-queries`
- I made a npm script to run it, `"groupcss" : "group-css-media-queries myinput.css myoutput.css"`
- I ran my npm script, `npm run groupcss`
- I opened the new file and compared with the source, and I saw no differences.

**Comment by ivankravchenko** [comment]
Should work :)
Could you provide sample of your input file?

**Comment by cyrfer** [comment]
Thank you for looking at the problem. I am attaching my input.

[input.zip](https://github.com/SE7ENSKY/group-css-media-queries/files/392896/input.zip)

**Comment by ivankravchenko** [comment]
I've run it locally with provided input, it turns this:
```css
@media (orientation: landscape) and (min-width: 415px) {
  .col-0-landscape {
    width: 0%
  }
}

@media (orientation: landscape) and (min-width: 415px) {
  .col-offset-0-landscape {
    margin-left: 0%
  }
}

@media (orientation: landscape) and (min-width: 415px) {
  .flex-item-h-0-landscape {
    width: 0%;
    flex-basis: 0%
  }
}
```
into
```css
@media (orientation: landscape) and (min-width: 415px) {
  .col-0-landscape {
    width: 0%;
  }

  .col-offset-0-landscape {
    margin-left: 0%;
  }

  .flex-item-h-0-landscape {
    width: 0%;
    flex-basis: 0%;
  }
  /* ... other rules with same media query... */
```

Please make sure you use latest 1.2.0 version.

**Comment by cyrfer** [comment]
It would be great if it did that for me. I am running on Windows 10, x64. Perhaps the implementation does not work on my OS.

I just noticed the version installed by NPM.
> "group-css-media-queries": "^1.1.1",

Why does the old version get installed?

**Comment by ivankravchenko** [comment]
There was a minor update yesterday.
Anyway, I expected this to work on Windows. I have no Windows at hand though. I will try my best to find one.

**Comment by cyrfer** [comment]
I updated. I still see no evidence that the tool does anything other than make a new copy of my file.

**Comment by mike3run** [comment]
Can confirm this works 100% on mac, maybe its a windows thing the bug 😢

**Comment by cyrfer** [comment]
I did more research. I have 2 projects, each using [Less](https://github.com/less/less.js) 2.7.1 and the Less [plugin](https://github.com/bassjobsen/less-plugin-group-css-media-queries) 1.1.1 which uses this project. Both projects use the same version of all dependencies, except for the Grunt version, and the same source .less files.

In the project that uses Grunt 0.4.5, I see desired behavior when I use `grunt less` to group media queries. This means that your code can run properly on my Windows computer.

In the project that uses Grunt 1.0.1, I see the same problems that I see when I run your code standalone, without Grunt.

Does this give you any ideas?

**Comment by cyrfer** [comment]
**I FOUND THE PROBLEM**
The code for css-parse is completely different in my 2 projects. How did a different version of the library get installed? I tracked down when the first `@media` rule was parsed in the different projects.

Here is what the code should look like:
![image](https://cloud.githubusercontent.com/assets/4392990/17835716/ee27e528-672d-11e6-9243-8452b797448d.png)

But here is what the code looks like in the project with the problem.
![image](https://cloud.githubusercontent.com/assets/4392990/17835723/2df512d4-672e-11e6-95a9-8830b402b9eb.png)

**Comment by cyrfer** [comment]
Please see this PR. It solved my problem.
https://github.com/SE7ENSKY/group-css-media-queries/pull/14

**Comment by ivankravchenko** [comment]
Please confirm if it's resolved in version 1.4.0 of `group-css-media-queries`.

**Comment by cyrfer** [comment]
@ivankravchenko Confirmed! Thank you for accepting the patch.
